src/datapoints-graph/charts/chart.model.ts

```diff
@@ -0,0 +1,30 @@
+import { SeriesValue } from '../model';
+
+export interface CustomSeriesOptions extends echarts.EChartsOption {
+  // typeOfSeries is used for formatter to distinguish between events/alarms series and datapoints
```

Comment:
"and datapoints", so datapoints are represented by `null`? :D

Reply:
changed the text :D